chromadb/server/fastapi/__init__.py

```diff
@@ -291,6 +293,12 @@ def app(self) -> fastapi.FastAPI:
     def root(self) -> Dict[str, int]:
         return {"nanosecond heartbeat": self._api.heartbeat()}
 
+    async def quota_exception_handler(request: Request, exc: QuotaError):
+        return JSONResponse(
+            status_code=429,
```

Comment:
Why is 429 (Too Many Requests) more appropriate than 413 (Payload Too Large)?

Reply:
we want something generic, and 429 is also use for resource exhausted. (gcp, aws)